Dockerfile

```
@@ -0,0 +1,26 @@
+FROM python:3.8.5-slim-buster
+
+RUN apt update && apt install -y \
```

```suggestion
RUN apt-get update && apt install -y \
```
You're supposed to use that command in scripts and similar, because the cli is more stable.